stdlib/private/StdlibUnicodeUnittest/UnicodeScalarProperties.swift

```diff
@@ -10,7 +10,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-// Unicode scalar tests are currently only avaible on Darwin, awaiting a sensible
+// Unicode scalar tests are currently only available on Darwin, awaiting a sensible
```

Comment:
This is well over 80 chars, so it'll have to be wrapped. (Is there tooling to scream at me?)